spaceship/static/wip-overlay.css

```diff
@@ -0,0 +1,78 @@
+#overlay-background {
+  background-color: rgb(0, 0, 0);
+  display: none;
+  left: 0px;
+  top: 0px;
+  height: 100%;
+  margin: 0px;
+  opacity: 0.75;
+  position: fixed;
+  width: 100%;
+  z-index: 999998;
+}
+.overlay-container {
+    box-sizing: border-box;
+    display: none;
+    margin: 0px auto;
+    padding: 15px 15px 15px 15px;
+    max-width: calc(100% - 30px);
+    position: fixed;
+    opacity: 1;
+    z-index: 999999;
+    left: 0px;
+    right: 0px;
+    width: 678px;
+    height: calc(100% - 20px);
+    top: 50%;
+    transform: translateY(-50%);
+    max-height: 510px;
+    background: rgba(255,255,255, 0.95);
+    h1,h2,h3,p,li,a {
+      font-family: "Circular Std", "Lato", sans-serif;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+      margin-top: 0;
+      padding-right: 7px;
+      padding-left: 7px;
+    }
+  }
+  #overlay .overlay-title {
+    font-family: 'Circular Std', 'Lato', sans-serif;
+    font-weight: bold;
+    font-size: 36px;
+    text-align: center;
+  }
+  #overlay .overlay-desc {
+    font-family: 'Circular Std', 'Lato', sans-serif;
+    font-weight: normal;
+    font-size: 18px;
+  }
+  #overlay .icon img{
+    width: 150px;
+    height: auto;
+    margin-bottom: 20px;
+  }
+  #overlay .close-button {
+    text-align: center !important;
```

Comment:
Yep !important is a really common symptom of a specificity issue. :) Here and below, though, I think it may actually not be necessary.

Reply:
👍